tests/unit/commands/test_list_validation_sets.py

```diff
@@ -0,0 +1,145 @@
+    )
+
+
+def test_list_validation_sets(click_run, fake_dashboard_get_validation_sets):
+    fake_dashboard_get_validation_sets.return_value = validation_sets.ValidationSets.unmarshal(
+        {
+            "assertions": [
+                {
+                    "headers": {
+                        "account-id": "AccountIDXXXOfTheRequestingUserX",
+                        "authority-id": "AccountIDXXXOfTheRequestingUserX",
+                        "name": "certification-x1",
+                        "revision": "222",
+                        "sequence": "9",
+                        "series": "16",
+                        "sign-key-sha3-384": "XSignXKeyXHashXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+                        "snaps": [
+                            {
+                                "id": "XXSnapIDForXSnapName1XXXXXXXXXXX",
+                                "name": "snap-name-1",
+                                "presence": "optional",
+                            },
+                            {
+                                "id": "XXSnapIDForXSnapName2XXXXXXXXXXX",
+                                "name": "snap-name-2",
+                            },
+                        ],
+                        "timestamp": "2020-10-29T16:36:56Z",
+                        "type": "validation-set",
+                    }
+                },
+                {
+                    "headers": {
+                        "account-id": "AccountIDXXXOfTheRequestingUserX",
+                        "authority-id": "AccountIDXXXOfTheRequestingUserX",
+                        "name": "acme-qa",
+                        "revision": "2",
+                        "sequence": "2",
+                        "series": "16",
+                        "sign-key-sha3-384": "XSignXKeyXHashXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+                        "snaps": [
+                            {
+                                "id": "XXSnapIDForXSnapName3XXXXXXXXXXX",
+                                "name": "snap-name-3",
+                                "presence": "required",
+                                "revision": "2",
+                            }
+                        ],
+                        "timestamp": "2020-10-29T16:36:56Z",
+                        "type": "validation-set",
+                    }
+                },
+            ]
+        }
+    )
+
+    cmd = ["list-validation-sets"]
+
+    result = click_run(cmd)
+
+    assert result.exit_code == 0
+    assert result.output == dedent(
+        """\
+        Account-ID                        Name              Sequence    Revision    When
+        AccountIDXXXOfTheRequestingUserX  certification-x1  9           222         2020-10-29
+        AccountIDXXXOfTheRequestingUserX  acme-qa           2           2           2020-10-29
+        """
+    )
+    fake_dashboard_get_validation_sets.assert_called_once_with(
+        name=None, sequence=None,
+    )
```

Comment:
Maybe add one more test for sequence=2 filter, or similar?

Reply:
I did that in the previous PR, this will only make its way to the mock (we pass on the string to the store as it holds the rule engine)